module/VuFind/src/VuFind/ChannelProvider/ChannelLoader.php

```diff
@@ -52,15 +53,15 @@ class ChannelLoader
     /**
      * Constructor
      *
-     * @param Config         $config         Channels configuration
+     * @param array          $config         Channels configuration
      * @param CacheManager   $cacheManager   Cache manager
      * @param ChannelManager $channelManager Channel manager
      * @param SearchRunner   $searchRunner   Search runner
      * @param RecordLoader   $recordLoader   Record loader
      * @param string         $locale         Current locale (used for caching)
      */
     public function __construct(
-        protected Config $config,
+        protected array $config,
```

Comment:
Now that the Config class is no longer used, we can remove the `use` statement for it above. There's a good chance that `composer fix` should do this for you automatically, though sometimes it's overly cautious if the class name appears as part of a comment -- so if it doesn't do the job, you'll have to remove it manually.